crates/next-api/src/module_graph.rs

```diff
@@ -512,6 +512,15 @@ async fn validate_pages_css_imports(
     graph.traverse_edges_from_entries(entries, |parent_info, node| {
         let module = node.module;
 
+        // If the module being imported isn't a global css module, there is nothing to validate.
+        let module_is_global_css =
+            ResolvedVc::try_downcast_type::<CssModuleAsset>(module).is_some();
+
+        if !module_is_global_css {
+            return GraphTraversalAction::Continue;
+        }
+
+        // We allow any global import of css files from the `node_modules` folder.
```

Comment:
This line is ambiguous:
- is it valid to import a CSS file that is inside of node_modules: `import "css-reset/style.css";`
- is it valid to import a CSS from a JS file in node_modules

You might be checking the wrong case here

Reply:
@mischnic What I'm referring to here is what the test cases enforce which is the first case that you describe. I don't see any tests anywhere regarding the second case, so I'm guessing not that. Is there a way to make that more clear?

Reply:
What I can say is that it correctly passes all of our tests that we have in place (which were apparently quite extensive haha), and there is no logic change here from the previous PR where this was introduced except in terms of order and trying to document it. I think all we need here is some clearer documentation to reduce the confusion you had. To be honest, I didn't even know that the second option was even a thing that was possible as I've never seen it attempted.

Reply:
That might unintentionally be allowed based on how it's implemented in Webpack, not sure.
---
```suggestion
        // We allow imports of global CSS files which are inside of `node_modules`.
```